keyboards/idobao/id87/v2/readme.md

```diff
@@ -0,0 +1,29 @@
+# ID87 v2
+
+![ID87 v2](https://i.imgur.com/woTSycN.jpg)
+
+A TKL keyboard with hotswap sockets and in switch RGB.
+
+* Keyboard Maintainer: [peepeetee](https://github.com/peepeetee)
+* Hardware Supported: ID87 v2
+* Hardware Availability: 
+  * https://drop.com/buy/idobao-id87-v2-tkl-mechanical-keyboard-kit
+  * https://idobao.net/products/idobao-id87-v2-tkl-pcb-mounted-hot-swappable-mechanical-keyboard-kit
```

Comment:
This may be better: 
```suggestion
* Hardware Availability: [Drop](https://drop.com/buy/idobao-id87-v2-tkl-mechanical-keyboard-kit), [idobao](https://idobao.net/products/idobao-id87-v2-tkl-pcb-mounted-hot-swappable-mechanical-keyboard-kit)
```